unit1-csharp/lesson03.rst

```diff
@@ -0,0 +1,45 @@
+Unit 1 - Lesson 3 Learning Objectives
+=====================================
+
+Goals
+-----
+
+- Work with strings using bracket notation and common string methods
+- Create and work with arrays using bracket notation and common array methods
+- Understand strings as ordered collections, similar to arrays
+
+Objectives
+----------
+
+Strings
+^^^^^^^
+
+- Understand strings, like arrays, are indexed collections
+- Index into a string using bracket notation
+- Describe what it means for strings to be immutable
+- Find the length of a string using the ``.Length`` method
+- Create strings with special characters: ``\n``, ``\t``, unicode (e.g. ``'\u03a9'``)
+- Include quotes with strings by escaping them
+- Use template literals to create strings that rely on variable values
+- Use common string methods: ``.IndexOf``, ``.ToCharArray``, ``.ToLower``, ``.ToUpper``, ``.Trim``, ``.Replace``, ``.Remove``
+- Understand how ASCII allows characters to be represented as numbers
+- Convert characters from string to ASCII codes and back using ``charCodeAt`` and ``String.fromCharCode``
+
+
+Arrays
+^^^^^^
+
+- Understand arrays, like strings, are indexed collections
+- Create an array literal
+- Create an array of size ``n`` with ``Array(n)``
+- Create arrays that store multiple types of values
+- Create arrays that store other arrays (i.e. multi-dimensional arrays)
+- Access elements within multi-dimensional arrays
+- Find the length of an array using the ``.Length`` method
+- Index into an array using bracket notation
+- Set an element of an array using bracket notation assignment
+- Describe what it means for arrays to be mutable
+- Understand what happens when asking for an element at index at which no element exists
+- Use common array methods: ``concat``, ``join``, ``split``, ``push``, ``pop``, ``shift``, ``unshift``, ``slice``, ``reverse``, ``sort``, ``indexOf``
+- Convert character arrays to strings using `.join(''s)`
```

Comment:
`.join(''s)` - the 's' here looks like a typo - is it?

Reply:
yes.  i will update my current version and hopefully correct that with the next push.